RoboJackets/urc-software
# Upgraded robot model

# Description
The robot model that was in the urc repo was only a basic model with no published sensor information. 

This PR does the following:
- Changes the robot model to look like a rocker bogie robot.
- Adds Sensors (left,right,center cameras, imu, gps, lidar) to the model.
- Adds a ground_truth topic to the model.

Fixes #14 

# Testing steps (If relevant)
## Test Case 1
1. go to colcon-urc, run colcon build --symlink-install and . install/setup.bash
2. run ros2 launch urc_gazebo simulation.launch.py
3. Make sure the robot model spawns in (might take a little) and make sure ros2 topic list has topics for all of the listed sensors.

Expectation: Robot model should load in and publish the added sensor topics

# Self Checklist
